classes/03class/exercises/c03-docker02/gpavelar/Dockerfile.v2

```diff
@@ -0,0 +1,3 @@
+FROM httpd:2.4.41-alpine
+RUN rm /usr/local/apache2/htdocs/index.html
+RUN ls /usr/local/apache2/htdocs
```

Comment:
you don't need this in the dockerfile. This is just a extra layer that doesn't add anything to the image itself

Reply:
Thanks, I kept it just to see if there is nothing inside the directory